Makefile

```diff
@@ -110,3 +110,9 @@ scan-vulnerability:
 	mkdir -p ./tmp/clair/usabillabv
 	cat ./tmp/build-*.tags | xargs -I % sh -c 'clair-scanner --ip 172.17.0.1 -r "./tmp/clair/%.json" -l ./tmp/clair/clair.log % || echo "% is vulnerable"'
 	docker-compose -f test/security/docker-compose.yml -p clair-ci down
+
+ci-scan-vulnerability:
+	docker-compose -f test/security/docker-compose.yml -p clair-ci up -d
+	RETRIES=0 && while ! wget -T 10 -q -O /dev/null http://localhost:6060/v1/namespaces ; do sleep 1 ; echo -n "." ; if [ $${RETRIES} -eq 10 ] ; then echo " Timeout, aborting." ; exit 1 ; fi ; RETRIES=$$(($${RETRIES}+1)) ; done
```

Comment:
:nail_care: It'd be much easier to read this bash script if it were spread out over multiple lines.

Reply:
Indeed, the whole script is quite similar to the `scan-vulnerability` target, so it should be doable

Reply:
@agustingomes It's exactly the same script, except it doesn't bring docker-compose down, plus it errors out when it finds a vulnerability.

@cvmiert While I agree, it's a literal port back of the removed lines. It also keeps the style for both `ci-scan-vulnerability` and `scan-vulnerability` the same. Plus I'm hoping to be able to replace it with Trivy turning it into a one liner without having to wait for docker-compose to come up: 

```Makefile
cat ./docker-image/image.tags | xargs -I % sh -c 'docker run -v /tmp/trivy:/var/lib/trivy -t aquasec/trivy:latest --cache-dir /var/lib/trivy image --exit-code 1 --no-progress --format table % || echo "% is vulnerable"'
```

Rerfs: https://github.com/WyriHaximusNet/docker-php/pull/129

So unless you really insist on improving the readability I'd prefer to keep it as is for now.

Reply:
> So unless you really insist on improving the readability I'd prefer to keep it as is for now.

Please move ahead :rocket: